src/style.css

```
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 4px;
+}
+
+.star--item {
+  height: 14px;
+  width: 13px;
+  background-image: url(images/star.svg);
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > .stars__star:first-child > .star--item,
+.stars--2 > .stars__star:nth-of-type(-n + 2) > .star--item,
+.stars--3 > .stars__star:nth-of-type(-n + 3) > .star--item,
+.stars--4 > .stars__star:nth-of-type(-n + 4) > .star--item,
+.stars--5 > .stars__star:nth-of-type(n) > .star--item {
+  background-image: url(images/star-active.svg);
+}
```

After optimization it will be like:

```suggestion
.stars--0 :nth-child(-n + 1),
.stars--1 :nth-child(-n + 2),
.stars--2 :nth-child(-n + 3),
.stars--3 :nth-child(-n + 4),
.stars--4 :nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}
```